src/lz4frame.jl

```diff
@@ -80,11 +89,16 @@ struct LZ4F_frameInfo_t
     blockChecksumFlag::Cuint
 end
 
-function LZ4F_frameInfo_t(; blocksizeid::UInt32=LZ4F_default, blockmode::Cuint=LZ4F_blockLinked, 
-    contentchecksumflag::Cuint=LZ4F_noContentChecksum, frametype::Cuint=LZ4F_frame, contentsize::Culonglong=(Culonglong)(0), 
-    blockchecksumflag::Cuint=LZ4F_noBlockChecksum) 
-
-    LZ4F_frameInfo_t(blocksizeid, blockmode, contentchecksumflag, frametype, contentsize, (UInt)(0), blockchecksumflag)
+function LZ4F_frameInfo_t(;
+    blocksizeid::BlockSizeID=default_size,
+    blockmode::BlockMode=block_linked,
+    contentchecksum::Bool=false,
+    frametype::FrameType=normal_frame,
+    contentsize::Int64=0, 
+    blockchecksum::Bool=false
+    ) 
```

Comment:
We usually would have this unindented one more level and then not have a blank space below